deploy/docker-ephemeral/build/Makefile

```diff
@@ -0,0 +1,206 @@
+USERNAME ?= julialongtin
```

Comment:
Could you rename this to e.g. `DOCKER_USERNAME`? It clashes with an env variable on my system.

Reply:
done.